restapi/api.go

```diff
@@ -37,7 +37,7 @@ func Router(db *gorm.DB, locationFinder location.LocationFinder) (http.Handler,
 		rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
 		rest.GetHandler(r, "/authors/origin_countries", api.ReturnAllCountries)
 		rest.PostHandler(r, "/login", api.Login)
-
+		rest.PostHandler(r, "/stories_formdata", api.CreateStoriesFormData)
```

Comment:
Can you put this under PrivateAPI we don't want people to add stories until we finish auth

Reply:
Should I put under Private after Auth is done. Might be easier to test and integrate with Frontend cuz i believe we are removing old auth anyways

Reply:
wait why are we calling the endpoint `stories_formdata`? Shouldn't we just be POSTing on `stories`?

Reply:
So i did not want to change the existing endpoint, which is why created a new one. Also the difference between these two is that `stories_formdata` (we can call it smth else or if we decide to get rid of the previous one, we can just call it stories) request payload is a formdata where as `stories` endpoint is a json. Also `stories_formdata` creates a single story, not sure how you would pass in a array of formdata. And looking at our use case, submitting a form on the Frontend would create just one story.

Reply:
I think we should just make the frontend send as `application/json` instead of form data, and make the endpoints JSON only.

Reply:
How would the FE pass in the image file in json (base64 encoding??) not sure how we would convert that in the backend but i can take a look at that if thats what we plan on doing. With form data, FE is just passing the image file directly.

Reply:
I mean the image file is just a byte array, so it can be accessed the same way in the JSON object iirc. 

I guess if it's too complicated we can leave it as is.

Reply:
`file, h, err := r.FormFile("image")` This is how it reads the file from formdata. If its json, it would not be like this